pennylane_lightning/src/gates/cpu_kernels/GateImplementationsLM.hpp

```diff
@@ -1004,6 +1097,61 @@ class GateImplementationsLM : public PauliGenerator<GateImplementationsLM> {
         }
     }
 
+    template <class PrecisionT>
+    static void applyCSWAP(std::complex<PrecisionT> *arr, size_t num_qubits,
```

Comment:
Coverage seems to drop here and below. Can we verify these are being tested?

Reply:
Thanks @mlxd ! In fact, as confirmed with @chaeyeunpark , the existing test kernels cover the newly implemented gates as shown [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/08831c19ea580e16998be8a94c5a1653575dfb5c/pennylane_lightning/src/tests/Test_GateImplementations_Nonparam.cpp#L618).